src/pages/Account/Venue/VenueMapEdition/Container.tsx

```diff
@@ -73,9 +74,15 @@ export const Container: React.FC<PropsType> = (props) => {
     backgroundImageStyle,
     containerStyle,
     lockAspectRatio,
+    isSaving,
   } = props;
   const [boxes, setBoxes] = useState<SubVenueIconMap>(iconsMap);
   const [imageDims, setImageDims] = useState<Dimensions>();
+  const [dragBoxId, setDragBoxId] = useState<number>(0);
+
+  const setDragItemId = useCallback((id: number) => {
+    setDragBoxId(id);
+  }, []);
```

Comment:
What does this function add that `setDragBoxId` doesn't already do by itself? Seems like a layer of indirection that adds no value?

Reply:
It's using `useCallback` that's it.

Reply:
`setDragBoxId` comes from `useState`, which is already a stable function reference. The only time wrapping it with `useCallback` makes sense is when we're 'closing around the state' to make a 'no arguments' function. But since we still have the `id` arg in the `useCallback` that is just passed through, there's no benefit to having this here.

We can remove this chunk of code and just call/pass `setDragBoxId` to the appropriate places.